docs/kusama-endpoints.md

```diff
@@ -0,0 +1,84 @@
+---
+id: kusama-endpoints
+title: Kusama Endpoints
+sidebar_label: Kusama Endpoints
+---
+
+When interacting with the [Kusama network](https://kusama.network) via [Polkadot Apps](https://polkadot.js.org/apps) or other UIs and programmatic methods, you'd ideally be running your own node ([text guide](maintain-sync), [video guide](https://www.youtube.com/watch?v=31DdfcxbAVs)). Granted, that's not something everyone can or wants to do, so convenience trumps ideals in most cases. To facilitate this convenience, Kusama has several public endpoints you can use for your apps.
+
+## Parity Archive Node
+
+The development company Parity maintains an archive node at endpoint `wss://kusama-rpc.polkadot.io/`. 
+
+To connect to the Parity node, use the endpoint in your JavaScript apps like so:
+
+```javascript
+const{ ApiPromise, WsProvider } = require('@polkadot/api')
+
+(async () => {
+    const provider = new WsProvider('wss://kusama-rpc.polkadot.io/')
+    const api = await ApiPromise.create({ provider })
+    // ...
+```
+
+or in Polkadot Apps UI by clicking the top-left corner of the screen and selecting the appropriate option:
+
+![Selecting Parity's Node](/img/endpoints/parity.png)
+
+## Web3 Foundation Archive Node
+
+The Web3 Foundation maintains an archive node at endpoint `wss://cc3-5.kusama.network/`. 
+
+To connect to this node, use the endpoint in your JavaScript apps like so:
```

Comment:
Could we just say "replace `'wss://kusama-rpc.polkadot.io/'` with `'wss://cc3-5.kusama.network/'`"?  Seems like including the rest of the code is superfluous here.